drivers/adc/ad4170/ad4170.c

```diff
@@ -0,0 +1,2116 @@
+		AD4170_REG_ADC_CHANNEL_MAP(7),
+		AD4170_REG_ADC_CHANNEL_SETUP(8),
+		AD4170_REG_ADC_CHANNEL_MAP(8),
+		AD4170_REG_ADC_CHANNEL_SETUP(9),
+		AD4170_REG_ADC_CHANNEL_MAP(9),
+		AD4170_REG_ADC_CHANNEL_SETUP(10),
+		AD4170_REG_ADC_CHANNEL_MAP(10),
+		AD4170_REG_ADC_CHANNEL_SETUP(11),
+		AD4170_REG_ADC_CHANNEL_MAP(11),
+		AD4170_REG_ADC_CHANNEL_SETUP(12),
+		AD4170_REG_ADC_CHANNEL_MAP(12),
+		AD4170_REG_ADC_CHANNEL_SETUP(13),
+		AD4170_REG_ADC_CHANNEL_MAP(13),
+		AD4170_REG_ADC_CHANNEL_SETUP(14),
+		AD4170_REG_ADC_CHANNEL_MAP(14),
+		AD4170_REG_ADC_CHANNEL_SETUP(15),
+		AD4170_REG_ADC_CHANNEL_MAP(15),
+		AD4170_REG_ADC_SETUPS_MISC(0),
+		AD4170_REG_ADC_SETUPS_AFE(0),
+		AD4170_REG_ADC_SETUPS_FILTER(0),
+		AD4170_REG_ADC_SETUPS_FILTER_FS(0),
+		AD4170_REG_ADC_SETUPS_OFFSET(0),
+		AD4170_REG_ADC_SETUPS_GAIN(0),
+		AD4170_REG_ADC_SETUPS_MISC(1),
+		AD4170_REG_ADC_SETUPS_AFE(1),
+		AD4170_REG_ADC_SETUPS_FILTER(1),
+		AD4170_REG_ADC_SETUPS_FILTER_FS(1),
+		AD4170_REG_ADC_SETUPS_OFFSET(1),
+		AD4170_REG_ADC_SETUPS_GAIN(1),
+		AD4170_REG_ADC_SETUPS_MISC(2),
+		AD4170_REG_ADC_SETUPS_AFE(2),
+		AD4170_REG_ADC_SETUPS_FILTER(2),
+		AD4170_REG_ADC_SETUPS_FILTER_FS(2),
+		AD4170_REG_ADC_SETUPS_OFFSET(2),
+		AD4170_REG_ADC_SETUPS_GAIN(2),
+		AD4170_REG_ADC_SETUPS_MISC(3),
+		AD4170_REG_ADC_SETUPS_AFE(3),
+		AD4170_REG_ADC_SETUPS_FILTER(3),
+		AD4170_REG_ADC_SETUPS_FILTER_FS(3),
+		AD4170_REG_ADC_SETUPS_OFFSET(3),
+		AD4170_REG_ADC_SETUPS_GAIN(3),
+		AD4170_REG_ADC_SETUPS_MISC(4),
+		AD4170_REG_ADC_SETUPS_AFE(4),
+		AD4170_REG_ADC_SETUPS_FILTER(4),
+		AD4170_REG_ADC_SETUPS_FILTER_FS(4),
+		AD4170_REG_ADC_SETUPS_OFFSET(4),
+		AD4170_REG_ADC_SETUPS_GAIN(4),
+		AD4170_REG_ADC_SETUPS_MISC(5),
+		AD4170_REG_ADC_SETUPS_AFE(5),
+		AD4170_REG_ADC_SETUPS_FILTER(5),
+		AD4170_REG_ADC_SETUPS_FILTER_FS(5),
+		AD4170_REG_ADC_SETUPS_OFFSET(5),
+		AD4170_REG_ADC_SETUPS_GAIN(5),
+		AD4170_REG_ADC_SETUPS_MISC(6),
+		AD4170_REG_ADC_SETUPS_AFE(6),
+		AD4170_REG_ADC_SETUPS_FILTER(6),
+		AD4170_REG_ADC_SETUPS_FILTER_FS(6),
+		AD4170_REG_ADC_SETUPS_OFFSET(6),
+		AD4170_REG_ADC_SETUPS_GAIN(6),
+		AD4170_REG_ADC_SETUPS_MISC(7),
+		AD4170_REG_ADC_SETUPS_AFE(7),
+		AD4170_REG_ADC_SETUPS_FILTER(7),
+		AD4170_REG_ADC_SETUPS_FILTER_FS(7),
+		AD4170_REG_ADC_SETUPS_OFFSET(7),
+		AD4170_REG_ADC_SETUPS_GAIN(7),
+		AD4170_REG_REF_CONTROL,
+		AD4170_REG_V_BIAS,
+		AD4170_REG_I_PULLUP,
+		AD4170_REG_CURRENT_SOURCE(0),
+		AD4170_REG_CURRENT_SOURCE(1),
+		AD4170_REG_CURRENT_SOURCE(2),
+		AD4170_REG_CURRENT_SOURCE(3),
+		AD4170_REG_FIR_CONTROL,
+		AD4170_REG_COEFF_WRITE_DATA,
+		AD4170_REG_COEFF_READ_DATA,
+		AD4170_REG_COEFF_ADDRESS,
+		AD4170_REG_COEFF_WRRD_STB,
+		AD4170_REG_DAC_SPAN,
+		AD4170_REG_DAC_CHANNEL_EN,
+		AD4170_REG_DAC_HW_TOGGLE_MASK,
+		AD4170_REG_DAC_HW_LDAC_MASK,
+		AD4170_REG_DAC_DATA(0),
+		AD4170_REG_DAC_SW_TOGGLE_TRIGGERS,
+		AD4170_REG_DAC_SW_LDAC_TRIGGERS,
+		AD4170_REG_DAC_INPUTA(0),
+		AD4170_REG_DAC_INPUTB(0),
+	};
+
+	uint32_t ri;
+	printf("[register]: [value]\n");
```

Comment:
let's go for `pr_info` (or even `pr_debug` in this particular case). same applies for all the other `printf` throughout the code.